L/louvain/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "louvain"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/KrainskiL/louvain.git", "9e3e081f054e531e32af38fe71d2991f954ea347")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd louvain/
+make
+mv hierarchy louvain matrix convert $prefix
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("hierarchy", :hierarchy),
+    ExecutableProduct("louvain", :louvain),
+    ExecutableProduct("convert", :convertlouvain)
+]
```

Comment:
Missing `matrix`?

Reply:
`matrix` executable is not needed in current release, but may be needed in the future. Still it stays in build script for convenience. Thank you for suggestions!